src/app/cli/src/explore/api_server.rs

```diff
@@ -51,11 +50,11 @@ impl APIServer {
     ) -> Result<Self, InternalError> {
         // Background task executor must run with server privileges to execute tasks on
         // behalf of the system, as they are automatically scheduled
-        let task_executor = cli_catalog.get_one().unwrap();
+        let task_executor = server_catalog.get_one().unwrap();
```

Comment:
Don't understand this change

Reply:
Now I see what you mean -- rolled it back